src/components/SearchResults.js

```diff
@@ -55,6 +55,13 @@ type SearchResultsProps = {
   showRatings: boolean;
   /** A style to apply to the results list */
   style: any;
+  /**
+   * A response can be passed in from custom searches if you don't want
+   *  to use the response on the searcher in this.context
+   */
+  response: QueryResponse | null;
+  /** Offset of search results for long scrolling or pagination */
+  offset: number;
```

Comment:
This should be made optional for backwards compatibility.
`offset?: number;`

Reply:
Done.